src/constants/blockchains.ts

```diff
@@ -1,4 +1,4 @@
-const TRANSACTION_TEMPLATE_ID_PLACEHOLDER = '{TRANSACTION_ID}';
+const TRANSACTION_TEMPLATE_ID_PLACEHOLDER: string = '{TRANSACTION_ID}';
```

Comment:
is there a reason to have 2 similar ones: https://github.com/blockchain-certificates/cert-verifier-js/pull/498/files#diff-9291e4734278c501ad261a1cde783f88R9?

Reply:
Ah yeah that's a mistake.